patches/net/minecraft/client/gui/screens/inventory/AbstractContainerScreen.java.patch

```diff
@@ -23,7 +15,12 @@
          ItemStack itemstack = this.draggingItem.isEmpty() ? this.menu.getCarried() : this.draggingItem;
          if (!itemstack.isEmpty()) {
              int l1 = 8;
-@@ -156,13 +_,16 @@
+@@ -152,17 +_,21 @@
+     @Override
+     public void renderBackground(GuiGraphics p_295206_, int p_295457_, int p_294596_, float p_296351_) {
+         super.renderBackground(p_295206_, p_295457_, p_294596_, p_296351_);
++        net.neoforged.neoforge.common.NeoForge.EVENT_BUS.post(new net.neoforged.neoforge.client.event.ContainerScreenEvent.Render.Background(this, p_295206_, p_295457_, p_295457_));
```

Comment:
This is still the wrong place, it should be after the `renderBg()` call

Reply:
fixed!